.github/workflows/deploy.yaml

```diff
@@ -386,13 +392,37 @@ jobs:
           deployment/seed-data.sh
         working-directory: .
 
+      - uses: actions/download-artifact@v3
+        with:
+          name: gaiaxkey
+          path: deployment/terraform/participant
+
+      - name: 'Push region verifiable credential'
+        run: |
+          mvn dependency:copy -Dartifact=org.eclipse.dataspaceconnector.identityhub:identity-hub-cli:$cli_version:jar:all -DoutputDirectory=.
+          
+          id=$(uuidgen)
+          
+          java -jar identity-hub-cli-$cli_version-all.jar -s=$identityHubUrl vc add \
```

Comment:
minor nit: if we're downloading the IH client, maybe give it a shorter name, e.g. `identityhub-cli.jar`?

Reply:
Since this part will be reworked in #32 I would let it be for the moment.